place_tracker/lib/place_details.dart

```diff
@@ -235,4 +246,88 @@ class _Map extends StatelessWidget {
       ),
     );
   }
+}
+
+class _Reviews extends StatelessWidget {
+
+  Widget _review(String reviewText) {
+    return Column(
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.symmetric(vertical: 10.0),
+          child: Row(
+            children: <Widget>[
+              Container(
+                width: 80.0,
+                height: 80.0,
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(40.0),
+                  border: Border.all(
+                    width: 3.0,
+                    color: Colors.grey,
+                  ),
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: <Widget>[
+                    const Text(
+                      '5',
+                      style: TextStyle(
+                          fontSize: 24.0,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.black87,
+                      ),
+                    ),
+                    const Icon(
+                      Icons.star,
+                      color: Colors.amber,
+                      size: 36.0,
+                    ),
+                  ],
+                ),
+              ),
+              const SizedBox(width: 16.0),
+              Expanded(
+                child: Text(
+                  reviewText,
+                  style: const TextStyle(fontSize: 20.0, color: Colors.black87),
+                  maxLines: null,
+                ),
+              ),
+            ],
+          ),
+        ),
+        Divider(
+          height: 8.0,
+          color: Colors.grey[700],
+        ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        const Padding(
+          padding: EdgeInsets.fromLTRB(0.0, 12.0, 0.0, 8.0),
+          child: Align(
+            alignment: Alignment.topLeft,
+            child: Text(
+              'Reviews',
+              style: TextStyle(
+                fontSize: 24.0,
+                fontWeight: FontWeight.bold,
+                decoration: TextDecoration.underline,
+                color: Colors.black87,
+              ),
+            ),
+          ),
+        ),
+        Column(
+          children: StubData.reviews.map((String reviewText) => _review(reviewText)).toList(),
```

Comment:
another nit: While flutter/flutter requires types on lambda parameters, I prefer to leave them out in our samples.